cli/tests/test_commit_command.rs

```diff
@@ -73,6 +73,27 @@ fn test_commit_with_editor() {
     "###);
 }
 
+#[test]
+#[cfg(windows)]
```

Comment:
nit: If the test passes on the other platforms, I would remove `cfg(windows)` (and move `use std::path::PathBuf` to top.) Windows-only code paths are easier to break.

Reply:
Done.